docs/book/user-guide/component-guide/model-deployers/bentoml.md

```diff
@@ -62,11 +62,11 @@ zenml model-deployer register bentoml_deployer --flavor=bentoml
 The ZenML integration will provision a local HTTP deployment server as a daemon process that will continue to run in the
 background to serve the latest models and Bentos.
 
-### How do you use it?
+## How do you use it?
 
-In order to use the BentoML Model Deployer, We need to understand these three main concepts:
+In order to use the BentoML Model Deployer, we need to understand three main concepts:
```

Comment:
```suggestion
In order to use the BentoML Model Deployer, you need to have an understanding of the following three concepts:
```

Reply:
Good point, the original wording was quite bad here. I rephrased this entirely 👍